src/elm/Utils.elm

```diff
@@ -7,13 +7,14 @@ port module Utils exposing
     , writeToLocalStorage
     , simpleUpdate
     , timeDiff
+    , toCmd
```

Comment:
thanks, i think this function was removed in https://github.com/parlez-vous/embed/pull/26/commits/1fbc82056f0a3a596da16ecffde40259141228da, please remove from here as well.

Reply:
Oops. My bad. Fixed it